apis/sqs/v1beta1/queue_types.go

```diff
@@ -51,20 +49,19 @@ type RedrivePolicy struct {
 	// The Amazon Resource Name (ARN) of the dead-letter queue to which Amazon
 	// SQS moves messages after the value of maxReceiveCount is exceeded.
 	// +optional
-	DeadLetterQueueARN *string `json:"deadLetterQueueARN,omitempty"`
+	DeadLetterTargetARN *string `json:"deadLetterTargetArn,omitempty"`
```

Comment:
This would be a breaking change to a `v1beta1` resource, but given that this field does not work currently I don't feel as though bumping the version would be required here. Does that sound right to you?

Reply:
Yes, that's why I didn't bump it.